scripts/build-post-list.js

```diff
@@ -58,10 +59,25 @@ function walkDirectories(directories, result, sectionWeight = 0, sectionTitle) {
         details.isIndex = fileName.endsWith('/index.md')
         details.slug = details.isIndex ? sectionSlug : slug.replace(/\.md$/, '')
         if(details.slug.includes('/specifications/') && !details.title) {
-          if(basename(data.slug).startsWith('v')) {
-            details.title = capitalize(basename(data.slug).slice(1))
+          let fileName = basename(data.slug)
+
+          if(fileName.includes('release')) {
+            details.isPrerelease = true
+            details.releaseDate = getReleaseDate(fileName)
+            fileName = fileName.split('-')[0]
+          }
```

Comment:
Some assumptions have been made here by me.

1. The rc will always have `release` in the filename.
2. The format for the filename will always be `<release-version>-<year>-<month>-<release>.md`. ex. `v2.1.0-2021-06-release.md`.
    This is being used in marking a rc as `pre-release` as well as getting the release date.

Reply:
good assumptions, and valid ones, you can put them as a comment for future generations and even refer https://github.com/asyncapi/spec/issues/513#issuecomment-825910521 in the comment for future generations